tests/test_yum_util.py

```diff
@@ -35,3 +36,15 @@ def test_invalid_config():
         assert not repo.is_valid()
     else:
         assert True
+
+
+def test_write_content(tmpdir):
+    test_content = 'test_content'
+    repo = YumRepo(
+        repourl='http://example.com/a/b/c/myrepo.repo', content=test_content,
+        dst_repos_dir=str(tmpdir)
+    )
+    repo.write_content()
+
+    with open(os.path.join(str(tmpdir), repo.filename)) as f:
```

Comment:
note: you could use dst_filename and drop the os dependency here.

Reply:
yes, however in this case at least the test tests if expected location was used.